README.md

```diff
@@ -108,6 +110,18 @@ with open('foo.md', 'r') as fin:
     rendered = mistletoe.markdown(fin, LaTeXRenderer)
 ```
 
+To reflow the text in a Markdown document with a max line length of 20 characters:
+
+```python
+import mistletoe
+from mistletoe.markdown_renderer import MarkdownRenderer
```

Comment:
Yeah, I think it's nice that you don't _have to_ import the markdown renderer (nor the latex renderer) if you don't want to. I don't mind an explicit import like this.

Maybe the module should be called `markdown` instead of `markdown_renderer`, for shorter imports? But that would break the symmetry with the other renderers. And renaming those modules would be a major compatibility break.

The one place where it would be nicer with a shorter reference path is in the CLI. I mean, a command like this isn't exactly easy to type:
```
mistletoe foo.md --renderer mistletoe.contrib.jira_renderer.JIRARenderer
```

But that could be solved in a different way; for example by letting the CLI know about at least the renderers in the mistletoe main package so that those could be referenced by name without the full path.

Reply:
Yeah, lots of topics for future improvements. :) The CLI and its options could definitely be improved. Naming probably as well (I don't like we don't historically follow strict camel case, i.e. `HtmlRenderer` for example). I have also thought about how to pass different renderers' options from the command line. Something that deserves a separate issue(s). That being said, I would leave it as-is for now, agreed? :)

Reply:
Yes, sounds good. Those are definitively separate issue(s) as you say.